Resources/Prototypes/Entities/Objects/Misc/paper.yml

```
@@ -15,6 +15,8 @@
       map: ["enum.PaperVisualLayers.Stamp"]
       visible: false
   - type: Paper
+    sound: !type:SoundCollectionSpecifier
+      collection: PaperScribbles
```

```suggestion
  - type: Paper
    sound:
      collection: PaperScribbles
```

type tag isnt needed here